microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -105,7 +105,9 @@ The _distance_ of an object in the JavaScript heap is the number of nodes on the
 
 The _shallow size_ is the size of the memory that is held by the object directly.
 
-The shallow size of an object is usually small, because JavaScript objects often only store the descriptions of the object in memory, and not their values. Arrays and strings are exceptions to this, and can have a significant shallow size. Most JavaScript objects store their values elsewhere in the renderer memory, and only expose a small wrapper object on the JavaScript heap.
+The shallow size of an object is usually small, because JavaScript objects often only store the descriptions of the object in memory, and not their values. Most JavaScript objects store their values elsewhere in the renderer memory, and only expose a small wrapper object on the JavaScript heap.
```

Comment:
```suggestion
The shallow size of an object is usually small, because JavaScript objects often only store the descriptions of the object in renderer memory, and not their values. Most JavaScript objects store their values elsewhere in the renderer memory, and only expose a small wrapper object on the JavaScript heap.
```
tech review this change

Reply:
The "one place" and the "other place" are both within the JS heap in the renderer memory.